src/khoj/processor/org_mode/org_to_jsonl.py

```diff
@@ -95,9 +95,12 @@ def extract_org_entries(org_files):
         entries = []
         entry_to_file_map = []
         for org_file in org_files:
-            org_file_entries = orgnode.makelist_with_filepath(str(org_file))
-            entry_to_file_map += zip(org_file_entries, [org_file] * len(org_file_entries))
-            entries.extend(org_file_entries)
+            try:
+                org_file_entries = orgnode.makelist_with_filepath(str(org_file))
```

Comment:
Can we put the `orgnode.makelist` used in `process_single_org_file` under a try/catch with logging as well?